### PR TITLE
chore: update changelog

### PR DESCRIPTION
**Pull request by hi-rustin**

### CODE REVIEW

**Review by hi-rustin — COMMENTED** [review]
🔢 Self-check

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,17 @@
...
+### Fixed
+
+- Do now allow empty spec name.
+- Bail out an error for non remote registry.
+
+### Changed
+
+- Add empty name case.
+- Add a case for inspect the unpublished workspace package.
+- All use ws in tests.
```
Comment (epage) [inline CHANGELOG.md:16]: Wouldn't these be more of "Internal" or something?

**Review by epage — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (hi-rustin) [reply CHANGELOG.md]: Removed!

**Review by hi-rustin — COMMENTED** [review]
🔢 Self-check

**Pull request merged**